### ISSUE TITLE
Feature request: Add a parallel view as in xiphos

### ISSUE THREAD

**Issue by LAfricain** [bibletime/bibletime#238]
It would be very nice to have a window with a parallel view, for comparing easily different version of the text, like in Xiphos:
![parallel_view](https://user-images.githubusercontent.com/19690477/82219947-4b52af00-991e-11ea-8ff3-331e139e7ad1.png)

**Comment by greg-hellings** [comment]
This already exists.

At the top of the Bible window, click the down arrow button beside the version name. From that menu navigate to "Add" and follow down through the submenus. You can add either Bibles or commentaries in parallel views with one another.

**Comment by LAfricain** [comment]
This is great thank you, but not easy to find!
